crates/bevy_ecs/src/world/mod.rs

```diff
@@ -3666,6 +3666,18 @@ impl World {
         schedules.allow_ambiguous_resource::<T>(self);
         self.insert_resource(schedules);
     }
+
+    /// Set the error handler to use for systems that return a [`Result`](crate::result::Result) in
+    /// a specific schedule.
+    pub fn set_schedule_error_handler(
```

Comment:
This doesn't feel particularly worthwhile as a method on `World`. I don't think users will be trying to access this data via `World` very often.

Reply:
I mostly followed `allow_ambiguous_component`, which also weaves the same method through the call stack.

That one also seems highly unlikely to be used at runtime, more than this one, I guess?

Happy to remove it to keep the API of `World` a bit cleaner.

Reply:
Never mind, I just realized I refactored this at some point and this method is no longer used anywhere. I'll remove it.

Reply:
Resolved in [95b54ba](https://github.com/bevyengine/bevy/pull/17753/commits/95b54bac8e63257562d3e5d6ef805455233e17bc).